src/config/chis-tg/config.json

```diff
@@ -10,7 +10,91 @@
       "domain": "moh-togo-uat.dev.medicmobile.org"
     }
   ],
-  "contact_types": [  
+  "contact_types": [
+    {
+      "name": "cg70_supervision",
+      "friendly": "Supervision Area",
+      "contact_type": "cg72_supervision",
+      "user_role": ["supervisor"],
+      "username_from_place": true,
+      "deactivate_users_on_replace": true,
+      "hierarchy": [
+        {
+          "friendly_name": "Sanitary Training",
+          "property_name": "sanitary_training",
+          "contact_type": "cf60_sanitary",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected Supervision Area",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(Supervision Area Name\\)", "\\(SUPERVISION AREA NAME\\)"],
+        "required": true
+      },
+      "place_properties": [
+        {
+          "friendly_name": "Supervision Area Name",
+          "property_name": "supervision_area",
+          "type": "string",
+          "parameter": ["\\(Supervision Area Name\\)", "\\(SUPERVISION AREA NAME\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "Supervision Area Full Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "{{ place.supervision_area }} ({{ lineage.sanitary_training }})",
+          "required": true
+        }
+      ],
+      "contact_properties": [
+        {
+          "friendly_name": "Last Name",
+          "property_name": "last_name",
+          "type": "string",
+          "parameter": ["\\(LAST NAME\\)", "\\(LASTNAME\\)", "\\(LAST-NAME\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "First Name",
+          "property_name": "first_name",
+          "type": "string",
+          "parameter": ["\\(Chw First Name\\)", "\\(Chw Firstname\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "Full Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "{{ contact.last_name }} {{ contact.first_name }}",
```

Comment:
in the contact forms, I'm seeing `First + Last` but here I'm seeing `Last + First`.
Elsewhere in the contact forms, I'm maybe seeing `Last + First` (I think CHW Post). Should this be consistent? Or is this on purpose?

Reply:
Yes this also should be [handle](https://github.com/medic/config-moh-togo/issues/254) in the forms. the first forms that was build in the project are using `First + Last`, the expected format is `Last + First`.